### ISSUE TITLE
Lean: current status of live data sources?

### ISSUE THREAD

**Issue by auwsom** [QuantConnect/Lean#4174]
Hi, I've searched on this, but last time I searched I know a lot was up in the air with Yahoo shutting down it's data source. 

Could anyone save me several hours of tinkering and tell me what the current status for data sources is for self-hosted Lean live or paper trading? Does the market data package from IB for free above a $2k account level work for Lean? 

Are the any other data streams from newer APIs like Alpaca or TD Ameritrade that I could work on converting the code for?

**Comment by jaredbroad** [comment]
Please post user application concerns to the [user-group](https://www.quantconnect.com/forum/discussions/1/lean) so we can keep the github focused on code development. 

That said - here's a [medium post](https://medium.com/hackernoon/setting-up-your-own-algorithmic-trading-server-4bbdf0766c17) on setting up your own algorithmic trading server

**Comment by auwsom** [comment]
@jaredbroad  Thanks very much for your reply. I want to say thank you for heading up the development of such a truly awesome platform since Quantopian died. (If it didn't sound unprofessional, I would say 'You are the man!'). 

I'm sure you have limited time, but most of my search results came from the user-group, but are often talking about QC webapp (which I'm too paranoid about IP loss to use, though I wish I could purchase data from you).. Is there a way to use the live data from a Prime Subscription in Lean?.. It looks like there is a config entry for QC live data, I may try that. I only make a few trades a year at present, so that may not fit my scenario, but I am willing to contribute to the development.

I searched the user-group directly on '[data live lean](https://www.quantconnect.com/forum/search/1/data+live+lean)', and although there isn't a Tag for Lean, and many of the results are out of date (and no date on each post heading), but I did find a reply (link below) implying that data could be used live in Lean. The [Medium post](https://medium.com/hackernoon/setting-up-your-own-algorithmic-trading-server-4bbdf0766c17) is about backtesting, and I've created the IEX API before but I believe it was only for live data. I don't know if that has changed, but there isn't an entry for IEX in the GitHub config.json file. I gave up on getting live data a couple years ago when I tried, after many days, weeks and months, so trying to avoid that this time. I will post a question on the user-group.

https://www.quantconnect.com/forum/discussion/4095/iex-live-data-feed-how-to-implement-it/p1
